components/furnituremix/application/src/app/core/contract/TenantConfiguration.ts

```diff
@@ -0,0 +1,22 @@
+export type PaymentImplementation = 'crystal' | 'stripe' | 'quickpay';
+export type CrystalFakePaymentImplementation = 'card' | 'coin';
+export type TenantConfiguration = {
+    currency: string;
+    logo?: {
+        key: string;
+        url: string;
+        variants?: Array<{
+            key: string;
+            url: string;
+            width: number;
+            height: number;
+        }>;
+    };
+    crystalPayments: CrystalFakePaymentImplementation[];
+    paymentImplementations: PaymentImplementation[];
+    paymentImplementationVariables?: {
```

Comment:
Why not `Record<keyof PaymentImplementation, Record<string,string>>`?
Also, the values of the variables can they not be non-string?

Reply:
`Record<PaymentImplementation, Record<string, string>>` right ? changed

Reply:
would be string only for now up to the next iteration where it could change